cmd/podman/varlink/io.podman.varlink

```diff
@@ -8,7 +8,7 @@ type Version (
   version: string,
   go_version: string,
   git_commit: string,
-  built: int,
+  built: string, # as RFC3339
```

Comment:
can you verify this still works with the remote client?

Reply:
Sure. Where can I find this client?